MTA-2239: Release Notes 7.0.3 (#874)
windup/windup-documentation: docs/topics/mta-rn-known-issues-7-0-3.adoc

// Module included in the following assemblies:
//
// * docs/release_notes/master.adoc

:_content-type: REFERENCE
[id="rn-known-issues-7-0-3_{context}"]
= Known issues

> This section provides known issues in {ProductFullName} version 7.0.3:

Author: ```suggestion
This section provides highlighted known issues in {ProductFullName} version 7.0.3:
```

> This part provides known issues present in {ProductFullName} version 7.0.3:

Author: ```suggestion
This section provides known issues in {ProductFullName} version 7.0.3:
```

// Module included in the following assemblies:
//
// * docs/release_notes/master.adoc

:_content-type: REFERENCE
[id="rn-known-issues-7-0-3_{context}"]
= Known issues

This part provides known issues present in {ProductFullName} version 7.0.3:

> //// 
> resolved
> .IntelliJ IDE plugin Keymap actions do not work
> 
> The IntelliJ IDE plugin Keymap actions do not work which prevents the {ProductShortName} automation. Currently, there is no workaround available. link:https://issues.redhat.com/browse/MTA-2460[(MTA-2460)]
> ////

Author: ```suggestion
```
